tavern/ent/schema/file.go

```diff
@@ -50,15 +47,14 @@ func (File) Fields() []ent.Field {
 			Comment("The timestamp for when the File was last modified"),
 
 		field.Bytes("content").
+			Annotations(
+				entgql.Skip(), // Don't return file content in GraphQL queries
+			).
 			Comment("The content of the file"),
 	}
 }
 
 // Edges of the File.
 func (File) Edges() []ent.Edge {
-	return []ent.Edge{
-		edge.From("deploymentConfigs", DeploymentConfig.Type).
-			Ref("file").
-			Comment("Deployment Configurations that depend on this file."),
-	}
+	return []ent.Edge{}
```

Comment:
Have the edges been removed here just since this PR is a clean slate or will we not being using edges in our models?

Reply:
Clean slate o7